### PR TITLE
M2-353: Select respondents

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

File: src/features/SelectRespondents/Select/Select.styles.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    font-weight: ${variables.font.weight.regular};
+    line-height: ${variables.lineHeight.md};
+    background-color: transparent;
+    border-radius: ${variables.borderRadius.xxxl};
+    padding: ${theme.spacing(0, 1.6)};
+
+    &.Mui-focused {
+      background-color: ${variables.palette.primary_alfa12};
+    }
+  }
```
Comment (kdanilenka32) [inline src/features/SelectRespondents/Select/Select.styles.ts:25]: pls add spaces

File: src/features/SelectRespondents/Select/Select.styles.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Box, styled, TextField } from '@mui/material';
+import { StyledBodyMedium } from 'styles/styledComponents/Typography';
```
Comment (kdanilenka32) [inline src/features/SelectRespondents/Select/Select.styles.ts:2]: pls fix spaces

File: src/features/SelectRespondents/SelectRespondents.const.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+export const options = [
+  { label: SearchAcross.all, value: SearchAcross.all },
+  { label: SearchAcross.selected, value: SearchAcross.selected },
+  { label: SearchAcross.unselected, value: SearchAcross.unselected },
+];
+
+export const getHeadCells = (
+  onSelectAllClick: (event: ChangeEvent<HTMLInputElement>) => void,
+  selectAllChecked: boolean,
+  t: TFunction,
```
Comment (kdanilenka32) [inline src/features/SelectRespondents/SelectRespondents.const.tsx:22]: you can import t func from i18n.ts file

File: src/features/SelectRespondents/SelectRespondents.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+  const handleFilterChange = ({ target: { value } }: ChangeEvent<HTMLInputElement>) => {
+    setSearchAcrossValue(value);
+  };
+
+  return (
+    <>
+      <StyledBodyMedium
+        dangerouslySetInnerHTML={{
+          __html: t('selectRespondentsDescription', {
```
Comment (kdanilenka32) [inline src/features/SelectRespondents/SelectRespondents.tsx:92]: Do you really need dangerouslySetInnerHTML?

File: src/features/SelectRespondents/Select/Select.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+import { SelectProps } from './Select.types';
+import { StyledTextField, StyledLabel, StyledSvgContainer } from './Select.styles';
+
+export const Select = ({ label, onChange, value, options }: SelectProps) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <StyledFlexTopCenter>
+      <StyledLabel>{t(label)}</StyledLabel>
```
Comment (kdanilenka32) [inline src/features/SelectRespondents/Select/Select.tsx:15]: Is it impossible reuse SelectController component?

File: src/components/Popups/Modal/Modal.styles.ts [hunk 6]
```diff
@@ -5,6 +5,7 @@ import theme from 'styles/theme';
 import { variables } from 'styles/variables';
 import { StyledClearedButton } from 'styles/styledComponents/ClearedButton';
 import { shouldForwardProp } from 'utils/shouldForwardProp';
+import { ActionsAlign } from 'components/Popups/Modal/Modal.types';
```
Comment (IhorNesterenko99) [inline src/components/Popups/Modal/Modal.styles.ts:8]: pls change the path to ./Modal.types

File: src/components/Tables/Table/Head/Head.types.ts [hunk 7]
```diff
@@ -6,4 +6,6 @@ export type HeadProps = {
   onRequestSort: (event: React.MouseEvent<unknown>, property: string) => void;
   order: Order;
   orderBy?: string;
+  headBackground?: string;
```
Comment (IhorNesterenko99) [inline src/components/Tables/Table/Head/Head.types.ts:9]: Maybe it's better to use ui type, as in previous Hanna's PR?
Could you pls take a look?

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**